src/exchanges.ts

```diff
@@ -295,6 +322,33 @@ export const supportedExchanges: Array<Exchange> = [
       findDefaultToken(CoinKey.SUSHI, ChainId.BSC),
     ],
   },
+  {
+    key: 'lif3swap-bsc',
+    name: 'Lif3 Swap',
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/exchanges/lif3.png',
+    chainId: 56,
+    webUrl: 'https://lif3.com/',
+    tokenlistUrl: 'https://assets.lif3.com/swap/bnb/tokens.json',
+    routerAddress: '0x1956005bA9690B7A00b09c78769675C0cE1457AF',
+    factoryAddress: '0x3FB1E7D5d9C974141A5B6E5fa4edab0a7Aa15C6A',
+    initCodeHash:
+      '0x57cf36086932d62d26c562d18d87afe373f166c06011dc35be0783e29c6eb4eb',
+    baseTokens: [
+      {
+        address: '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c',
+        symbol: 'WBNB',
+        chainId: ChainId.BSC,
+        decimals: 18,
+      } as Token,
+      {
+        address: '0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d',
+        symbol: 'USDC',
+        chainId: ChainId.BSC,
+        decimals: 18,
+      } as Token,
```

Comment:
If this is the default USDC, we should rather do `findDefaultToken(CoinKey.USDC, ChainId.BSC),